test_project/index.js

```diff
@@ -0,0 +1,17 @@
+const electron = require('electron');
```

Comment:
What is the overall purpose of the `test_project`. Is it used to run tests somewhere or am I missing something?